### ISSUE TITLE
CI: Link to a `demo.rerun.io` for each PR

### ISSUE THREAD

**Issue by emilk** [rerun-io/rerun#2167]
We have a nice `PR Build Summary` for each of our PRs, but still a lot of web-related bugs slip through.

I think a good next step is to include a link to `app.rerun.io` (or `demo.rerun.io`, or `test.rerun.io`) directly in the PR description, with a checkbox, e.g.:

* [ ] I have tested https://app.rerun.io/commit/be4014c/index.html

And then we mandate that this i checked before we can merge.

**Comment by emilk** [comment]
Actually, it would probably be better to keep `demo.rerun.io` user-facing, and instead publish to e.g. `test.rerun.io` or similar

**Comment by jprochazk** [comment]
Should we create a new issue for part 2? (`test.rerun.io`)

**Comment by jprochazk** [comment]
I created a separate issue, because `test.rerun.io` is a separate task (splitting `demo.rerun.io` into "fast examples to test with" and "something to show off")
